query-engine/connectors/mongodb-query-connector/src/interface/connection.rs

```diff
@@ -98,20 +98,17 @@ impl WriteOperations for MongoDbConnection {
         .await
     }
 
-    async fn execute_raw(
```

Comment:
Can probably be even more radical (not in scope here, just a thought): Only have one raw api function on the connector, the "handle all raw input" can be enough for the connector to dispatch itself.